exporter/metrics_test.go

```diff
@@ -227,6 +227,10 @@ func initGaugeRegex(t *testing.T) *oniguruma.Regex {
 }
 
 func newMetricConfig(t *testing.T, cfg *configuration.MetricConfig) *configuration.MetricConfig {
+	// Handle default for counter's value
+	if len(cfg.Value) == 0 {
```

Comment:
Use `if metric.Type == "counter" && len(metric.Value) == 0` here so that we have the same line as in `configV3.go`.

Reply:
Agreed.

Reply:
On a second note, disagreed, reverted in 4eddad3.
The metrics configuration which is seeded in, does not contain the metrics type.

Reply:
Oh, ok, I wasn't aware of that.